src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -806,7 +806,7 @@ public List<CustomNodeDefinition> CustomNodeDefinitions
             }
         }
 
-        private Dictionary<string, string> CustomDyfFilepaths { get; set; } = new Dictionary<string, string>();
+        internal Dictionary<string, string> CustomDyfFilepaths { get; set; } = new Dictionary<string, string>();
```

Comment:
We need to make this internal so we can access it from inside the `PackageManagerClientViewModel`